Lib/fontmake/instantiator.py

```diff
@@ -396,7 +396,7 @@ def generate_instance(
                 # whatever reason (usually outline incompatibility)...
                 if glyph_name not in self.skip_export_glyphs:
                     raise InstantiatorError(
-                        f"Failed to generate instance of glyph '{glyph_name}': "
+                        f"Failed to generate instance of glyph {glyph_name}: "
```

Comment:
here we lost the quotes, you meant to do this

```diff
+ f"Failed to generate instance of glyph {glyph_name}: "
- f"Failed to generate instance of glyph {glyph_name!r}: "
```

Reply:
Yeah, same as above. Fixed.